testdata/baselines/reference/conformance/deepElementAccessExpressionInJS.js

```diff
@@ -0,0 +1,24 @@
+//// [tests/cases/conformance/salsa/deepElementAccessExpressionInJS.ts] ////
+
+//// [declarations.d.ts]
+declare var module: {
+    exports: {
+        [key: string]: any;
+    };
+}
+//// [elementAccessExpressionInJS.js]
+if (module[calculatePropertyName(1)]) {
+}
+function calculatePropertyName(index) {
+    // this would be some webpack index in real life
+    return `property${index}`;
+}
+
+
+//// [elementAccessExpressionInJS.js]

```

Comment:
Something is wrong here; these are not declaration files, these are JS files.

Reply:
For sure, double js emit and no declaration file seems pretty wrong.